src/tests/run.py

```diff
@@ -376,7 +357,7 @@ def __create_bash_wrapper__(self):
     echo \"CORE_ROOT set to ${CORE_ROOT}\"
 fi
 
-""" % (self.unique_name, self.core_root)
+""" % (self.unique_name, self.args.core_root)
```

Comment:
This fixes an actual bug, which indicates that nobody ever ran this script on Linux!